docs/static/examples/development.yaml

```diff
@@ -106,7 +106,7 @@ write_files:
         echo "Helm installed!"
 
         echo "Adding helm repo..."
-        helm repo add united-manufacturing-hub https://repo.umh.app/
+        helm repo add united-manufacturing-hub https://deploy-preview-554--umh-helm-repo.netlify.app/ 
```

Comment:
Is this link static for all further development versions of the umh stack ?

Reply:
This will be changed once I went through the final testing (see also the update process). We can keep this conversation open until the final merge.